src/core/ext/transport/chttp2/transport/stream_map.cc

```diff
@@ -74,7 +82,7 @@ void grpc_chttp2_stream_map_add(grpc_chttp2_stream_map* map, uint32_t key,
     } else {
       /* resize when less than 25% of the table is free, because compaction
          won't help much */
-      map->capacity = capacity = 3 * capacity / 2;
+      map->capacity = capacity = 2 * capacity;
```

Comment:
what's the reason for the change?

Reply:
(Slightly) smaller number of ops when growing, and you'll need to grow less often since you grow a bit more.